ui/src/main/webapp/src/app/reports/execution-application-list/execution-application-list.component.ts

```diff
@@ -48,6 +48,11 @@ export class ExecutionApplicationListComponent extends RoutedComponent implement
     ngOnInit(): any {
         this.addSubscription(this.flatRouteLoaded.subscribe(flatRouteData => {
             let executionId = parseInt(flatRouteData.params['executionId']);
+            if (isNaN(executionId)) {
+                console.error("executionId is not set.", flatRouteData);
+                throw new Error(ExecutionApplicationListComponent.name + ": executionId is not set.");
```

Comment:
Where is `ExecutionApplicationListComponent.name` variable defined?

Reply:
That's the name of the class. ES6 feature.